eng/Versions.props

```diff
@@ -72,6 +72,7 @@
     <!-- for templates -->
     <MicrosoftExtensionsHttpResiliencePackageVersionForNet8>$(MicrosoftExtensionsHttpResiliencePackageVersion)</MicrosoftExtensionsHttpResiliencePackageVersionForNet8>
     <MicrosoftExtensionsHttpResiliencePackageVersionForNet9>$(MicrosoftExtensionsHttpResiliencePackageVersion)</MicrosoftExtensionsHttpResiliencePackageVersionForNet9>
```

Comment:
What's the story with `MicrosoftExtensionsHttpResiliencePackageVersionForNet9`? It has the same version as "ForNet8". Shouldn't these 2 versions be separate, so when I get a .NET 9 app, I get the 9.0 Resilience package?

Reply:
Yep, it's listed in #6076 as a TODO

Reply:
I'm going to address the remaining content items in #6076 as part of the next PR which refactors the templates down to a single template you can select TFM and Aspire version from.